src/backend/api/menu.ts

```diff
@@ -35,3 +35,5 @@ export const isAddedToSteam = async (appName: string, runner: Runner) =>
   ipcRenderer.invoke('isAddedToSteam', appName, runner)
 export const isEosOverlayEnabled = async (appName?: string) =>
   ipcRenderer.invoke('isEosOverlayEnabled', appName)
+export const runGogSetupForGame = async (appName: string) =>
```

Comment:
Same here, now that we *can* add return types to IPC functions, I'd say we should